### PR TITLE
Add Pr practice

### PR DESCRIPTION
**Pull request by Kylin-1q84**
Juwon is hell chang

### CODE REVIEW

**Comment by wotmd** [comment]
Juwon is hell chang

**Comment by illian01** [comment]
Hell galler

File: index.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <html>
-	<head>IOI 홈페이지</head>
-	<div>IOI 사진</div>
-	<body>IoI is beautiful</body>
```
Comment (illian01) [inline index.html]: IOI가 있었군!

Comment (Kylin-1q84) [reply index.html]: IoI !

**Pull request merged**